recipes/easyexif/all/conanfile.py

```diff
@@ -1,5 +1,9 @@
 import os
-from conans import CMake, ConanFile, tools
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get
+
 
 class EasyExifConan(ConanFile):
```

Comment:
since this uses `rm_Safe`

```suggestion

required_conan_version >= "1.53.0"


class EasyExifConan(ConanFile):
```